modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/sidebar/DataAndViewsTab.es.js

```diff
@@ -105,6 +110,7 @@ export default function DataAndViewsTab({
 	dispatch,
 	dispatchConfig,
 }) {
+	const {objectsPortletURL} = useContext(AppContext);
```

Comment:
Hey @gabrielpramos can you move the `const {objectsPortletURL} = useContext(AppContext)` statement close to its use?

Thx